R/recipe.R

```diff
@@ -209,6 +209,10 @@ recipe.formula <- function(formula, data, ...) {
     cli::cli_abort("{.arg data} is missing with no default.")
   }
 
+  if (is.table(data)) {
```

Comment:
Should we add a warning or deprecation notice for this? I'd like to get rid of this down the road; it shouldn't be supported.

Reply:
sure, i'm gonna add a warning and remove this in the following release with https://github.com/tidymodels/recipes/issues/1416